lib/active_admin/resource_controller/decorators.rb

```diff
@@ -57,8 +57,6 @@ def self.wrap(decorator)
           @cache[name] ||= wrap! collection_decorator, name
         end
 
-        private
```

Comment:
Same here, it seems like the code wanted to make `wrap!` private, but it doesn't work like that.